src/Devices/TempProbe_TC.h

```diff
@@ -0,0 +1,116 @@
+// Adafruit MAX31865
```

Comment:
Please convert this to a multi-line comment to allow it to show in documentation.